src/desdemona/tasks/kafka.clj

```diff
@@ -2,6 +2,12 @@
   (:require [taoensso.timbre :refer [info]]
             [cheshire.core :as json]))
 
+(defn deserialize-message-json [bytes]
+  (try
+    (json/parse-string (String. bytes "UTF-8"))
+    (catch Exception e
+      {:error e})))
```

Comment:
We don't do anything with this error right now, right? It's not clear to me what the Onyx best practice here is. I could certainly see how we would want to log these, for example...